packages/code-connect/components/DualListSelector/DualListHeader.figma.tsx

```diff
@@ -0,0 +1,21 @@
+import figma from '@figma/code-connect';
+import { DualListSelectorPane } from '@patternfly/react-core';
+
+/**
+ * PatternFly DualListSelectorHeader component integration for Figma Code Connect
+ */
+
+figma.connect(
+  DualListSelectorPane,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=21279-116172&m=dev',
+  {
+    props: {
+      // string
+      headerText: figma.string('Header text'),
+      itemInformation: figma.string('Item information'),
+
+      children: figma.children('*')
+    },
+    example: (props) => <DualListSelectorPane title={props.headerText}>{props.children}</DualListSelectorPane>
```

Comment:
I think the item information can be passed as the `status` prop? that's where you'd write like "0 of 3 options selected" for example

Reply:
It might be wise to wrap the children in a <DualListSelectorList> component as well, if that is not somewhere else...